### ISSUE TITLE
improve mobile views

### ISSUE THREAD

**Issue by Heklaterriol** [jemproject/JEM-Project#1645]
On smaller screens the event list and the modules are not very well formatted, they sometime look very chaotic – the response css definitely needs an improvement.

**Comment by jojo12** [comment]
This is for later, please! We finally wait for 4.01 as bugfix!

**Comment by jojo12** [comment]
Do you really mean the frontend? Where?

**Comment by Heklaterriol** [comment]
> This is for later, please! We finally wait for 4.01 as bugfix!

Yes of course for later!
Thank you for the clarification and setting the milestone.

**Comment by Heklaterriol** [comment]
> Do you really mean the frontend? Where?

<img width="375" alt="IMG_4516" src="https://github.com/jemproject/JEM-Project/assets/28540581/e1da900a-64a9-4b28-b24c-861966fbdb81">


I will create and post an suggestion of displaying it. Eg. I would suggest, that the colums switch to rows …

**Comment by jojo12** [comment]
If someone needs all in portrait, it will be difficult, for all we have the details view.
I can't reproduce in cassiopeia:
![grafik](https://github.com/jemproject/JEM-Project/assets/2960236/ea1139d4-7f45-49a7-84a3-92356799fcf7)

**Comment by Heklaterriol** [comment]
Yes, if you have only three columns, then it is fine. If you have more (like in my screenshot) then it looks ugly.

We could find a solution, where the table layout is dependent on the number of displayed columns.

**Comment by mckillo** [comment]
Closing this issue because the original general responsive/mobile layout problem has largely been addressed in the current JEM codebase with responsive templates and CSS for the main views and modules.

**Comment by mckillo** [comment]
Reviewed against the current JEM 4.5.0 codebase. The original broad mobile/responsive layout issue has been addressed with responsive templates and CSS for the main frontend views and modules. Event lists now have responsive/list layouts that handle many columns better on small screens.

This issue is closed.
